models/selective_triggers.go

```diff
@@ -359,6 +380,34 @@ func boolPtrValue(item map[string]any, key string) (*bool, error) {
 
 	return &boolValue, nil
 }
+func priorityValue(item map[string]any) (*int, error) {
+	valuePtr, err := intPtrValue(item, "priority")
+	if err != nil {
+		return nil, err
+	}
+	if valuePtr == nil {
+		return nil, nil
+	}
+
+	if *valuePtr > 100 || *valuePtr < -100 {
+		return nil, fmt.Errorf("priority (%d) should be between -100 and 100", *valuePtr)
+	}
+	return valuePtr, nil
```

Comment:
This validation is the same as the workflow priority validation. Could we extract it and share it between the two? It will help us avoiding changing only one of them if we ever want to change it in the future. 😄

Reply:
Updated in https://github.com/bitrise-io/bitrise/pull/1056/commits/a67405ceefa315d0d6519215dcbe26f2eedc30f4